samples/data-sealing/common/dispatcher.cpp

```diff
@@ -173,17 +173,24 @@ int ecall_dispatcher::unseal_data(
     size_t* data_size)
 {
     oe_result_t result = OE_OK;
-    unsigned char iv[IV_SIZE];
+    unsigned char iv[sealed_data_size];
```

Comment:
Consider keeping the original line since non-constant array sizes are not supported by all compilers.